backstop/tests/carousel.html

```diff
@@ -483,9 +483,8 @@ <h2>Default</h2>
 
         // initialize
         updateSlide(0);
```

Comment:
This seems to cause issues as it takes time to add required classes. I am thinking to just add those classes in HTML initially.

Reply:
Honestly, I have only seen flakiness in `Carousel` tests once or twice, so I'm not even sure about removing autoplay.

What's a lot more annoying is this `Tile` one which keeps failing in every PR:
![image](https://user-images.githubusercontent.com/9084735/154121129-ecae3dac-02be-4299-b417-7c17c164d45c.png)

Reply:
That screenshot is from this very PR. The last commit failed.

I also noticed the same failure in your previous PR (treenode).